sdk/ai/Azure.AI.Inference/api/Azure.AI.Inference.net8.0.cs

```diff
@@ -92,6 +92,7 @@ public ChatCompletionsClient(System.Uri endpoint, Azure.AzureKeyCredential crede
         public ChatCompletionsClient(System.Uri endpoint, Azure.AzureKeyCredential credential, Azure.AI.Inference.AzureAIInferenceClientOptions options) { }
         public ChatCompletionsClient(System.Uri endpoint, Azure.Core.TokenCredential credential) { }
         public ChatCompletionsClient(System.Uri endpoint, Azure.Core.TokenCredential credential, Azure.AI.Inference.AzureAIInferenceClientOptions options) { }
+        public ChatCompletionsClient(System.Uri endpoint, Azure.Core.TokenCredential credential, string[] authorizationScopes, Azure.AI.Inference.AzureAIInferenceClientOptions options = null) { }
```

Comment:
I don't think we want to add this ctor. Users will not know what to pass to scopes. Also, if they pass wrong scope (or use ctor that does not take it) the error is something impossible to understand.